kolibri/plugins/facility_management/assets/src/views/user-table.vue

```diff
@@ -1,11 +1,14 @@
 <template>
 
-  <!-- TODO MOVE THIS COMPONENT OUT AFTER STRINGFREEZE -->
   <div>
     <core-table class="user-table">
-      <caption class="title">
+      <caption v-if="title" class="title">
```

Comment:
To my eye, the role badge appears a little too close to the user's name. Maybe make it a little bit bigger (8-12px).

Reply:
Added between role badge and name. Which did you want bigger?